galaxy_ng/app/api/v3/serializers/collection.py

```diff
@@ -142,7 +136,7 @@ def to_internal_value(self, data):
             filename_tuple = parse_collection_filename(filename)
         except ValueError as exc:
             errors['filename'] = _get_error_details(exc, default_code='invalid')
-            log.error('CollectionUploadSerializer validation of filename failed: %s',
+            log.error(_('CollectionUploadSerializer validation of filename failed: %s'),
```

Comment:
Log entries doesn't need translation as it is not exposed to the user, logs are more useful for devs and ops so it is better to search for occurrences in English.